README.md

```diff
@@ -1,54 +1,129 @@
-# Technology
+
+# ChallengeGo
+
+O ChallengeGo é uma API para um simples sistema bancário, com os seguintes objetivos:
+
+* Criar uma conta;
+* Autenticar-se;
+* Realizar depósitos;
+* Realizar transferências;
+* Realizar saques;
+
+A API acessa um banco de dados em MySQL, onde são mantidas duas tabelas:
+* account
+* transfer
+
+O banco é versionado com migrate. Ao fazer o deploy da API, as tabelas serão criadas. Para alterar as tabelas, versionando o banco, podem ser executados os comandos:
+
+
+```bash
+
+migrate -path gateway/db/mysql/migration -database "mysql://root:password@tcp(localhost:3306)/sys?multiStatements=true" -verbose up
+
+```
+
+<h2>Rotas da aplicação</h2>
+
+A API está documentada com swagger que se encontra na url http://localhost:8080/docs/swagger/
+
+Exemplo de criação de conta:
+
+http://localhost:8080/accounts<br>
+
+Body json:
+
+```javascript
+{
+    "nome": "Fran",
+    "cpf":"98711222334",
+    "secret": "123"  
+}
+```
+Para efetuar login e gerar o token, que permitirá o acesso às rotas autenticadas (/transfer, /deposit, /withdraw), é preciso passar as credenciais, conforme o exemplo:
+
+http://localhost:8080/auth<br>
+
+```javascript
+{
+    "cpf":"98711222334",
+    "secret": "123"  
+}
+```
+
+# Tecnologias
+
 * Go 1.16.5
+
 * Docker 🐳
+
 * MySQL 5.7 🐬
 
-# Library
+* Clean Architecture
+
+  
+
+# Bibliotecas
+
```

Comment:
I think it would be nice to have links to the libs here, WDYT?

Reply:
yes, i agree!